### PR TITLE
[Validator] use a valid example

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/Validator/Tests/Constraints/CountValidatorTest.php [hunk 1]
```diff
@@ -209,12 +209,12 @@ public function testConstraintAnnotationDefaultOption()
     public function testDivisibleBy()
     {
         $constraint = new Count([
-            'divisibleBy' => 123,
+            'divisibleBy' => 3,
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/Tests/Constraints/CountValidatorTest.php]: The validated collection contains three elements. Checking whether it is divisible by 123 would result in a constraint validation error.

Comment (fancyweb) [reply src/Symfony/Component/Validator/Tests/Constraints/CountValidatorTest.php]: It was intentional to highlight the fact that the `divisibleBy` option is not really tested here, ie that the value does not matter in this test.

Comment (fabpot) [reply src/Symfony/Component/Validator/Tests/Constraints/CountValidatorTest.php]: Let's close then? Or perhaps we can add a comment to explain?

Comment (fancyweb) [reply src/Symfony/Component/Validator/Tests/Constraints/CountValidatorTest.php]: There is already a comment above the test. I will let you decide what is the best here.

**Pull request closed without merging**